recipe/meta.yaml

```diff
@@ -140,8 +139,6 @@ outputs:
         # Putting these here means they get emitted to build_env_setup.{sh,bat} meaning we can launch IDEs
         # after sourcing or calling that script without examine the contents of conda_build.{sh,bat} for
         # important env. vars.
-        - OPENSSL_DIR=%PREFIX%\Library    # [win]
```

Comment:
This seems to be suspicious. For us, on anaconda, those lines seem still be required.  Why it works on your side without them?

Reply:
You might not have the latest version of the patch https://github.com/conda-forge/python-feedstock/blob/071818b32099122e18735c6c7a40591250bca4c5/recipe/patches/0009-Unvendor-openssl.patch

Reply:
I applied this and it works now, thanks